cmd/livepeer/livepeer.go

```diff
@@ -106,6 +106,9 @@ func main() {
 	gsBucket := flag.String("gsbucket", "", "Google storage bucket")
 	gsKey := flag.String("gskey", "", "Google Storage private key file name (in json format)")
 
+	// API
+	authWebhookURL := flag.String("authWebhookUrl", "", "Authentication webhook URL")
```

Comment:
Can the message mention that it's a "RTMP authentication webhook URL" ? That makes it clearer what it's used for.

Reply:
done